lib/workers/repository/updates/generate.spec.ts

```diff
@@ -304,6 +304,30 @@ describe('workers/repository/updates/generate', () => {
       expect(res.recreateClosed).toBeTrue();
     });
 
+    it('Grouped pin & pinDigest cannot be recreated when closed if recreateWhen is never', () => {
```

Comment:
I don't understand this test description. How about something like this, where we move the `If` statement to the front of the sentence, and then explain what happens:

> If recreateWhen is set to never: grouped pin & pinDigest updates are not recreated after the PR is closed